docs/source/calling-smart-contracts.md

```diff
@@ -85,6 +85,22 @@ def calling_other_contract() -> str:
     return neo_symbol
 ```
 
+### Calling NEP-17 Contracts
+In addition, Neo3-Boa has an interface for contracts that adhere to the NEP-17 standard.
+```python
+# calling_nep17_contract.py
+from boa3.sc.compiletime import public
+from boa3.sc.contracts import ContractManagement
+from boa3.sc.types import Nep17Contract
+
+
+@public
+def calling_other_contract(my_contract_hash: str) -> str:
+    nep_17_contract: Nep17Contract = ContractManagement.get_contract(my_contract_hash)
```

Comment:
I think this is going to throw compilation errors. `get_contract` argument should me `UInt160` shouldn't it?
If this is the case, change the variable type
```suggestion
from boa3.sc.types import Nep17Contract, UInt160


@public
def calling_other_contract(my_contract_hash: UInt160) -> str:
    nep_17_contract: Nep17Contract = ContractManagement.get_contract(my_contract_hash)
```